resources/license_banner.txt

```diff
@@ -0,0 +1,2 @@
+FingerprintJS Pro GTM v<%= pkg.version %> - Copyright (c) FingerprintJS, Inc, <%= new Date().getFullYear() %> (https://fingerprint.com)
```

Comment:
I'm not sure that we need a banner.

Reply:
Sure, I included all resources I found in various `resources` folders across the repositories, assuming they are all needed somehow. But if this one is not used I will delete it :)